rpm/SPECS/contextBroker.spec

```diff
@@ -117,8 +117,10 @@ rm $RPM_BUILD_ROOT/usr/share/contextBroker/tests/cases/CMakeLists.txt
 cp LICENSE $RPM_BUILD_ROOT/usr/share/doc/contextBroker
 cp scripts/testEnv.sh test/functionalTest/testHarness.sh test/functionalTest/testDiff.py test/functionalTest/harnessFunctions.sh $RPM_BUILD_ROOT/usr/share/contextBroker/tests
 cp scripts/accumulator-server.py $RPM_BUILD_ROOT/usr/share/contextBroker/tests 
+cp scripts/httpsPrepare.sh $RPM_BUILD_ROOT/usr/share/contextBroker/tests
```

Comment:
I guess the easiest way to fix this is to call httpsPrepare.sh using $PATH.
We'll all need to add "~/git/fiware-orion/scripts" (or wherever we have the repo) to our PATH.
The problem arises when we use more than one repo (not that httpsPrepare.sh will change between repos, but still).

Another way would be to create an ENV VAR pointing to the file.

No good way of solving this, really